packages/flutter/lib/src/cupertino/default_text_selection_toolbar.dart

```diff
@@ -0,0 +1,205 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart' show defaultTargetPlatform;
+import 'package:flutter/widgets.dart';
+
+import 'desktop_text_selection_toolbar.dart';
+import 'text_selection_toolbar.dart';
+import 'text_selection_toolbar_buttons_builder.dart';
+
+/// The default Cupertino context menu for text selection for the current
+/// platform.
+///
+/// Builds the mobile Cupertino context menu for all mobile platforms, not just
```

Comment:
Then maybe it needs a different name that doesn't include `Cupertino`?

Reply:
I've been putting al macOS stuff into the Cupertino library and calling it Cupertino, but maybe that's not accurate.  Should we come up with another name that encompasses both iOS and macOS?